### PR TITLE
Create docker-tags.yaml

### PR DESCRIPTION
**Pull request by pascalwilbrink**
## Hey, I just made a Pull Request!
I've created a plugin that displays the Docker tags for a specific component

#### :heavy_check_mark: Checklist

I don't think the next checklist is applicable :)

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/docker-tags.yaml
Comment (awanlin) [inline microsite/data/plugins/docker-tags.yaml:7]:
```suggestion
documentation: 'https://github.com/Work-m8/backstage-docker-plugin/blob/main/README.md'
```

This get's you the raw Markdown, you probably want to use this 👆

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for the contribution @pascalwilbrink, left a comment about the documentation link

Would also be great to get an image of what this feature looks like in use in your docs so that potential users have a feel for the features

**Comment by pascalwilbrink** [comment]
@awanlin, Thanks for you suggestions :D I've updated the yaml file and added some screenshots in the README.md

**Review by awanlin — APPROVED** [review]
Looking good @pascalwilbrink 🚀 

We just need a review and approval from one of the maintainers now

**Comment by pascalwilbrink** [comment]
Looks like a flaky gh action

File: microsite/data/plugins/docker-tags.yaml
Comment (tudi2d) [inline microsite/data/plugins/docker-tags.yaml:4]: Do you actually want to redirect to the repository? Or the organisation? :)

```suggestion
authorUrl: 'https://github.com/work-m8'
```

**Review by tudi2d — APPROVED** [review]
Welcome to the community & thanks for adding yourself to the Plugins Directory! 🎉

**Comment by pascalwilbrink** [comment]
Any updates on this?

**Review by freben — APPROVED** [review]

**Pull request merged**
